introduction/intro.md

```diff
@@ -13,166 +13,30 @@ redirect_from:
 
 How does Qubes OS provide security?
------------------------------------
-
-Qubes takes an approach called **security by compartmentalization**, which
-allows you to compartmentalize the various parts of your digital life into
-securely isolated compartments called *qubes*.
-
-This approach allows you to keep the different things you do on your computer
-securely separated from each other in isolated qubes so that one qube getting
-compromised won't affect the others. For example, you might have one qube for
-visiting untrusted websites and a different qube for doing online banking. This
-way, if your untrusted browsing qube gets compromised by a malware-laden
-website, your online banking activities won't be at risk. Similarly, if
-you're concerned about malicious email attachments, Qubes can make it so
-that every attachment gets opened in its own single-use [disposable
-qube]. In this way, Qubes allows you to do everything on the same physical
-computer without having to worry about a single successful cyberattack taking
-down your entire digital life in one fell swoop.
-
-Moreover, all of these isolated qubes are integrated into a single, usable
-system. Programs are isolated in their own separate qubes, but all windows are
-displayed in a single, unified desktop environment with [unforgeable colored
-window borders][getting started] so that you can easily identify windows from
-different security levels. Common attack vectors like network cards and USB
-controllers are isolated in their own hardware qubes while their functionality
-is preserved through secure [networking], [firewalls], and [USB device
-management][USB]. Integrated [file] and [clipboard] copy and paste operations
-make it easy to work across various qubes without compromising security. The
-innovative [Template] system separates software installation from software use,
-allowing qubes to share a root filesystem without sacrificing security (and
-saving disk space, to boot). Qubes even allows you to sanitize PDFs and images
-in a few clicks. Users concerned about privacy will appreciate the
-[integration][Qubes-Whonix] of [Whonix] with Qubes, which makes it easy to use
-[Tor] securely, while those concerned about physical hardware attacks will
+------------------------------------
+
+Behind the scenes, Qubes OS leverages virtualization and more specifically the [open-source Xen hypervisor](https://wiki.xen.org/wiki/Xen_Project_Software_Overview) to allow the creation and management of well-isolated virtual machines called *qubes*. Those qubes, which are also referred simply to as *domains* or *compartments*, have specific :
+
+* **Purposes** : for personal or professional projects, to manage the USB or network stack.
+* **Natures** : full-fledged or stripped-down virtual machines which can be based on Fedora, Debian or even Windows.
```

Comment:
"based on popular operating systems such as Fedora, Debian, or even Windows". this makes it more clear that there are other OS  templates as well.

Reply:
Done